init_query.sql

```diff
@@ -40,22 +43,59 @@ LEFT JOIN
 ON
     word.speaker_id = t.speaker_id
 LEFT JOIN
-	phonestats as ps -- number of phones per word
+	phones_per_word
 ON
-	phone.wd_id = ps.wd_id
+	phone.wd_id = phones_per_word.wd_id
 LEFT JOIN
-	formstats as fs -- word frequency
+	forms
 ON
-	word.cldf_name = fs.cldf_name AND
-	word.cldf_languageReference = fs.cldf_languageReference
+	word.cldf_name = forms.form AND
+	word.cldf_languageReference = forms.cldf_languageReference
 LEFT JOIN
     utterances AS utt  -- utterance-level stats such as speech rate.
 ON
     phone.u_ID = utt.u_id
-LEFT JOIN
-	sdev AS sd
+LEFT JOIN -- summary stats on word length per language
+    (
+        SELECT
+            stdev(p.num_phones) AS num_phones,
+            AVG(p.num_phones) AS avg_num_phones,
+            w.cldf_languageReference
+        FROM
+            phones_per_word as p
+        LEFT JOIN
+            'words.csv' AS w
+        ON
+            p.wd_id = w.cldf_id
+        GROUP BY
+            w.cldf_languageReference
+    ) AS sd_num_phones
+ON word.cldf_languageReference = sd_num_phones.cldf_languageReference
```

Comment:
@Tarotis I think including the summary stats like this is fairly transparent. The input data is clear from `FROM` and `LEFT JOIN`, the grouping is clear from the `GROUP BY` clause.